runtime/jcl/common/jdk_internal_misc_ScopedMemoryAccess.cpp

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+
+#include "jni.h"
+#include "jcl.h"
+#include "jclglob.h"
+#include "jclprots.h"
+#include "jcl_internal.h"
+
+extern "C" {
+
+#if JAVA_SPEC_VERSION >= 16
+void JNICALL
+Java_jdk_internal_misc_ScopedMemoryAccess_registerNatives(JNIEnv *env, jclass clazz)
+{
+	/* TODO: Add implementation. */
+}
+
+jboolean JNICALL
+Java_jdk_internal_misc_ScopedMemoryAccess_closeScope0(JNIEnv *env, jobject instance, jobject scope, jobject exception)
+{
+	/* TODO: Add implementation. */
+	return JNI_FALSE;
```

Comment:
Is this native actually invoked? `assert(!"Java_jdk_internal_misc_ScopedMemoryAccess_closeScope0 is unimplemented")` can indicate this is a stub, otherwise, an issue could be opened to track the actual implementation.

Reply:
The assertion has been added. The implementation of the stubs and associated test failures are tracked through https://github.com/eclipse/openj9/issues/11027 **(JEP393)**. Refer to the latest update for more details: https://github.com/eclipse/openj9/issues/11027#issuecomment-730849265.

1. `ScopedMemoryAccess_registerNatives`: should not assert; stays empty since our tooling handles the registration of Java native methods.
2. `ScopedMemoryAccess_closeScope0`: now, throws an assertion; needs an implementation.